pom.xml

```diff
@@ -752,18 +750,6 @@
           </exclusion>
         </exclusions>
       </dependency>
-      <dependency>
-        <groupId>xerces</groupId>
-        <artifactId>xercesImpl</artifactId>
-        <version>${xercesImpl.version}</version>
-        <scope>test</scope>
-        <exclusions>
```

Comment:
Shouldn't we add the exclusions in `core/pom.xml` if we're removing them here?
The parent POM dependency management doesn't have these exclusions.